packages/headache/headache.1.04/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+name: "headache"
+version: "1.04"
+
+license: "GNU Library General Public License"
+
+synopsis: "Automatic generation of files headers"
+description: """
+Lightweight tool for managing headers in source code files. It can
+update in any source code files (OCaml, C, XML et al).
+"""
+
+authors: [
+  "Vincent Simonet"
+  # contributors
+  "Patrick Baudin"
+  "Mehdi Dogguy"
+  "François Pottier"
+  "Virgile Prevosto"
+  "Ralf Treinen"
+]
+
+maintainer: "Patrick Baudin"
+homepage: "https://github.com/Frama-C/headache/"
+bug-reports: "https://github.com/Frama-C/headache/issues"
+dev-repo: "git+https://github.com/Frama-C/headache.git"
+
+depends: [
+  "camomile"
+  "dune" {build}
+]
+
+build: [
+  [ "dune" "build" "-p" "headache" ]
```

Comment:
The more standard way of calling dune build is `"dune" "build" "-p" name "-j" jobs`. `"headache"` is replaced by `name` which is more generic and avoid to do mistakes if it gets renamed or copy-paste and `"-j" jobs` is useful for people wanting to let dune have the full power of their CPU cores or on the contrary limit to be able to do other task while it's installing

Reply:
Of course, let's go for the standard way...